src/main/java/Duke.java

```diff
@@ -1,29 +1,37 @@
+import java.util.ArrayList;
 import java.util.Scanner;
 
 public class Duke {
-    public static final String LINE = "____________________________________________________________";
+    private static final String LINE = "____________________________________________________________";
+
+    private static ArrayList<String> tasks = new ArrayList<String>();
+
     public static void main(String[] args) {
         printWelcome();
 
         boolean isExit = false;
         while (!isExit) {
             String fullCommand = readUserCommand();
             System.out.println(LINE);
-            System.out.println("Duke: Your input is \"" + fullCommand + "\".");
-            System.out.println(LINE);
+
             switch(fullCommand) {
             case "bye":
                 isExit = true;
                 break;
+            case "list":
+                listTasks();
+                break;
             default:
-                //Do nothing
+                addTask(fullCommand);
+                System.out.println("Duke: Your input is \"" + fullCommand + "\".");
+                System.out.println(LINE);
             }
```

Comment:
I think this part is what I need to learn. Acutally I tried to use the switch method in my code but I don't know why it can't work well, make me use a long if else condition to make the program work.